source/includes/markdown/_07-auth.md

```diff
@@ -229,6 +239,50 @@ To implement the Authorization Code Grant flow (the most typical flow for most a
 
 The token that you have at the end can be used to make calls to the Asana API on the user's behalf.
 
+### Proof Key for Code Exchange (PKCE)
+
+> User Authorization Endpoint
```

Comment:
Since these aren't the only parameters used when making these two requests, how do you feel about calling out that these are the PKCE-specific params? Something like "Parameters used for PKCE with the User Authorization Endpoint" (I know we mention them up above in the properties, but I'm thinking about folks who might follow a link directly here and not see those 🤔 downside: it's definitely more wordy.)